### PR TITLE
[TUBEMQ-219] [website] Fix missing web title

### PR DESCRIPTION
**Pull request by cku328**
I configured the metadata of title for each markdown page.

### CODE REVIEW

File: docs/en-us/architecture.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+title: Architecture - Apache TubeMQ
+---
```
Comment (guangxuCheng) [inline docs/en-us/architecture.md:3]: image

We already have a title "TubeMQ Architecture". Why add a new title?

Comment (cku328) [reply docs/en-us/architecture.md:3]: Thanks @guangxuCheng  for the comment.

This title is a metadata item, it is used in DocSite generation page (from markdown .md file), it is not displayed on page (html).
```
For markdown documents, docsite provides three metadata configuration items:
title, keywords, and description. At the same time, the corresponding jsx page 
is rendered into static HTML and combined into the final generated page.
```
Ref. https://docsite.js.org/en-us/docs/SEO.html

Comment (guangxuCheng) [reply docs/en-us/architecture.md:3]: got that.

Comment (wf123537200) [reply docs/en-us/architecture.md:3]: i think the title should not include ```Apache TubeMq```,it may be a trouble to remind all docs to add the title

Comment (cku328) [reply docs/en-us/architecture.md:3]: @wf123537200 Do you mean the title doesn't need to include `Apache TubeMQ` suffix?

Comment (cku328) [reply docs/en-us/architecture.md:3]: If adding title metadata to the document is cumbersome, an even more extreme approach is to modify the template file to define the title tag directly as a specified string like "Apache TubeMQ".

I think that would solve this issue as well, but I don't think it's appropriate considering the search engine optimization.

What do you guys think?

Comment (guangxuCheng) [reply docs/en-us/architecture.md:3]: Metadata title is docsite's syntax not markdown, it is easy for contributors or reviewers to ignore this title when adding new document which will lead to inconsistent document formats on website. In addition, whether or not to add this title will not affect the user's reading. 

I just spoke out my worries, anyway, +1 for the pr

**Comment by cku328** [comment]
rebased latest master-branch (#36 / commit: ec0ab0c) & conflict resolved.

**Pull request merged**